aptos-move/block-executor/src/value_exchange.rs

```diff
@@ -19,6 +19,7 @@ use move_core_types::value::{IdentifierMappingKind, MoveTypeLayout};
 use move_vm_runtime::AsFunctionValueExtension;
 use move_vm_types::{
     delayed_values::delayed_field_id::{DelayedFieldID, ExtractWidth, TryFromMoveValue},
+    gas::AlwaysVisitedModuleTraversalContext,
```

Comment:
for some reason I don't like these names 🤦‍♂️ maybe Noop + Immutable?

Reply:
Didn't really know how to call them😃 I like noop and immutable 👍

Reply:
Renamed to `NoOpTraversalContext` and `ImmutableTraversalContext`.